Add configuration files for the new snapshot creation disruption scenario

LibinLiu opened the pull request:
To add the configuration files for the new snapshot creation disruption scenario.

Commit:
feat(cnv): Add configuration for snapshot disruption scenario

This commit adds the YAML configuration file for the new chaos scenario,
`snapshot_creation_disruption`. This scenario is designed to test the
resilience of the CNV snapshotting mechanism.

The new configuration file is structured to be clear, intuitive, and
robust, following the latest project best practices.

Key Features of This New Configuration:

1.  **Focused Execution Model:**
    - The configuration targets pre-existing Virtual Machines exclusively,
      simplifying the scenario's scope to focus purely on chaos injection.

2.  **Self-Documenting and Structured:**
    - The YAML file (`snapshot_creation_disruption.yaml`) is extensively
      documented with comments for every parameter, including default values.
    - Parameters are logically grouped and ordered to reflect the
      execution flow, making it easy to understand and use.

3.  **Flexible Timeouts:**
    - Includes configurable timeout values (e.g., `vmi_wait_timeout`,
      `snapshot_ready_timeout`) to allow easy adaptation to different
      cluster environments.

4.  **Clean Directory Structure:**
    - The new configuration is placed in a flattened directory hierarchy
      at `cnv/` to maintain a simple and clean structure.

Note: This scenario configuration is designed to work with the
corresponding plugin implementation in `krkn-chaos/krkn#988`.

Signed-off-by: Libin Liu <liliu@redhat.com>

LibinLiu commented:
Hi @paigerube14 I've now finished to refactor the code based on krkn-lib functions and your previous comments, and committed the cnv configuration yaml files update, could you please have a review? If any issues or comments, please just let me know. Thanks!

LibinLiu commented:
Hi @paigerube14 I've also flattened the directory structure by removing the lp_chaos level to simplify the hierarchy, could you please have a review? If any issues or comments, please just let me know. Thanks!

paigerube14 reviewed (approved):
/lgtm

paigerube14 commented:
before merging, are you able to [squash your commits ](https://krkn-chaos.dev/docs/contribution-guidelines/git-pointers/#squash-commits)into 1 and signoff on your one commit using instructions here: https://github.com/krkn-chaos/scenarios-hub/pull/2/checks?check_run_id=59982947135

LibinLiu commented:
> before merging, are you able to [squash your commits ](https://krkn-chaos.dev/docs/contribution-guidelines/git-pointers/#squash-commits)into 1 and signoff on your one commit using instructions here: https://github.com/krkn-chaos/scenarios-hub/pull/2/checks?check_run_id=59982947135

Hi @paigerube14 , thanks for your review!

I've squashed the commits into a single one and added the sign-off as you requested. The PR should be ready to merge now. Thanks!
